pkgs/by-name/st/steam-play-none/package.nix

```diff
@@ -0,0 +1,39 @@
+{ fetchFromGitHub
+, stdenvNoCC
+, lib
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  name = "Steam-Play-None";
+  version = "0-unstable-2022-12-15";
+  src = fetchFromGitHub {
+    repo = finalAttrs.name;
+    owner = "Scrumplex";
+    rev = "42e38706eb37fdaaedbe9951d59ef44148fcacbf";
+    hash = "sha256-sSHLrB5TlGMKpztTnbh5oIOhcrRd+ke2OUUbiQUqoh0=";
+  };
+  outputs = [ "out" "steamcompattool" ];
+  buildCommand = ''
+    runHook preBuild
+
+    # Make it impossible to add to an environment. You should use the appropriate NixOS option.
+    # Also leave some breadcrumbs in the file.
+    echo "${finalAttrs.name} should not be installed into environments. Please use programs.steam.extraCompatPackages instead." > $out
+
+    ln -s $src $steamcompattool
+
+    runHook postBuild
+  '';
+
+  meta = {
+    description = ''
+      Steam Play Compatibility Tool to run games as-is
+
+      (This is intended for use in the `programs.steam.extraCompatPackages` option only.)
+    '';
+    homepage = "https://github.com/Scrumplex/Steam-Play-None";
+    license = lib.licenses.cc0;
+    maintainers = with lib.maintainers; [ matthewcroughan scrumplex ];
+    platforms = [ "x86_64-linux" ];
```

Comment:
Technically this runs anywhere where bash exists ^^

Reply:
Let's also keep this for now, I don't think the script has much use on aarch64-linux or darwin.